### DISCUSSION TITLE
--link for 3.10.0

### DISCUSSION THREAD

**Discussion by Slodziak190q** [coder/code-server#4460] [Ideas]
Jm a code-server raspberry pi user and j want to forward my code-server but using ngrok is not very safe beacouse it don't have random IPs can you add - - link as another package on npm or uptade code-server3.10.0?

**Comment by lg-nineoneninethreenine** [comment]
It doesn't work sonce you need Coder Cloud, which is naturally available on the installable files.

**Reply by Slodziak190q** [reply]
Hi @im-coder-lg can you tell me where's the code-server files placed after installing it with npm?

**Reply by lg-nineoneninethreenine** [reply]
Idk but try `which code-server`, it might give the file location.

**Reply by lg-nineoneninethreenine** [reply]
Yeah I guess it's `/usr/bin/code-server`, might be different for you, type `which code-server` on LXTerminal(or any other terminal emulator or even PuTTY), that might locate your installation.

**Reply by lg-nineoneninethreenine** [reply]
Looks kinda like this:

![which code-server result](https://user-images.githubusercontent.com/76845820/140756508-b1ccad1e-04d0-4599-8fe7-62f3816012fe.png)

PS: Screenshot taken while running Xrdp, so the terminal might differ for you. But the command is the same. And it was taken on a Raspberry Pi.
